Lib/test/test_tools/test_msgfmt.py

```diff
@@ -121,10 +124,59 @@ def test_nonexistent_file(self):
         assert_python_failure(msgfmt, 'nonexistent.po')
 
 
+class Test_multi_input(unittest.TestCase):
```

Comment:
To follow the naming convention in this file:
```suggestion
class MultiInputTest(unittest.TestCase):
```